### PR TITLE
Add SD Events

### PR DESCRIPTION
**Pull request by askpatrickw**
South Dakota:  "The Mount Rushmore State"

image

### CODE REVIEW

File: states/south_dakota/elections.toml [hunk 1]
```diff
@@ -3,12 +3,41 @@ date = 2020-06-02
...
+# original_date = 2020-08-11
+# Not Needed this election cycle
 
 [20201103]
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+recieved_by = 2020-10-19T17:00:00
```
Comment (tannewt) [inline states/south_dakota/elections.toml]:
```suggestion
received_by = 2020-10-19T17:00:00
```

File: states/south_dakota/elections.toml [hunk 2]
```diff
@@ -3,12 +3,41 @@ date = 2020-06-02
...
+sources = [
+    "https://sdsos.gov/elections-voting/assets/2020ElectionCALENDAR.pdf",
+    "https://sdsos.gov/elections-voting/voting/register-to-vote/default.aspx",
+    "https://sdsos.gov/elections-voting/upcoming-elections/general-information/2020-Election-Information.aspx"
+]
+
+[20201103.absentee.application]
+in_person_by = 2020-08-10T17:00:00
+recieved_by = 2020-08-10T17:00:00
```
Comment (tannewt) [inline states/south_dakota/elections.toml]:
```suggestion
received_by = 2020-08-10T17:00:00
```

File: states/south_dakota/elections.toml [hunk 3]
```diff
@@ -3,12 +3,41 @@ date = 2020-06-02
...
+sources = [
+    "https://sdsos.gov/elections-voting/assets/2020ElectionCALENDAR.pdf",
+    "https://sdsos.gov/elections-voting/voting/absentee-voting.aspx"
+]
+
+[20201103.absentee]
+in_person_starts = 2020-09-18
```
Comment (tannewt) [inline states/south_dakota/elections.toml:34]: Sure this is in person? seems like this is by mail

**Review by tannewt — CHANGES_REQUESTED** [review]
Thank you! No rush on this.

File: states/south_dakota/elections.toml (hunk 3 above)
Comment (askpatrickw) [reply states/south_dakota/elections.toml:34]: Hmm... kind of unclear 
"Registered voters can vote in-person once Absentee Voting begins at their County Auditor's office by bringing along a valid photo identification card (ID). If a voter does not have a photo ID, they must be given the option to sign a personal identification affidavit and vote a regular ballot. "

So are they showing up and doing an in person absentee or is it early polling? Hard to say, they kind of mingle the processes into one.

**Comment by askpatrickw** [comment]
More changes in the last push.

File: states/south_dakota/elections.toml (hunk 3 above)
Comment (tannewt) [reply states/south_dakota/elections.toml:34]: Ah, in person makes sense in this case. I was just thinking it was mailed out.

**Review by tannewt — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**